ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/issues/TestIssue35.java

```diff
@@ -0,0 +1,40 @@
+package ye.weicheng.ngbatis.demo.issues;
+
+import com.alibaba.fastjson.JSON;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import ye.weicheng.ngbatis.demo.pojo.Person;
+import ye.weicheng.ngbatis.demo.repository.TestRepository;
+
+/**
+ * https://github.com/nebula-contrib/ngbatis/issues/35.
+ * @author yeweicheng
+ * @since 2022-09-01 20:20
+ * <br>Now is history!
+ */
+@SpringBootTest
+public class TestIssue35 {

```

Comment:
Unit tests should be functional, and a specific problem can be used as a use case in the unit test.

please add license for a new file.